README.md

```diff
@@ -1,26 +1,36 @@
-Bitcoin Core integration/staging tree
+Namecoin Core integration/staging tree
 =====================================
 
-[![Build Status](https://travis-ci.org/bitcoin/bitcoin.svg?branch=master)](https://travis-ci.org/bitcoin/bitcoin)
+https://namecoin.info/
```

Comment:
I think we're pushing the use of namecoin.org instead of namecoin.info now, so this should probably be changed to that.